openapi.yaml

```diff
@@ -1,394 +1,418 @@
+          description: "Status of passport to get"
+          required: true
+          schema:
+            type: "boolean"
       responses:
         "200":
           description: "Successful operation"
           content:
-            '*/*' :
+            '*/*':
               schema:
-                type: object
-                items:
-                  $ref: "#/components/schemas/Passport"
+                $ref: "#/components/schemas/PassportResponse"
         "400":
           description: "Invalid Order"
     put:
       tags:
-      - "passportStore"
-      summary: "Updates a passport"
-      description: "Updates passport"
+        - "PassportStore"
+      summary: "Update passport"
+      description: "Update passport by ID"
       operationId: "updatePassport"
       parameters:
-      - name: "number"
-        in: "path"
-        description: "number of passport to get"
-        required: true
-        schema:
-          type: "integer"
-          format: "int64"
+        - name: "id"
+          in: "path"
+          description: "ID of passport to update"
+          required: true
+          schema:
+            type: "string"
       requestBody:
-        content: 
+        content:
           'application/json':
             schema:
-              $ref: '#/components/schemas/Passport'
-          'application/xml':
-            schema:
-              $ref: '#/components/schemas/Passport'
+              $ref: '#/components/schemas/PassportRequest'
       responses:
         "200":
           description: "Successful operation"
           content:
-            '*/*' :
+            '*/*':
               schema:
-                type: object
-                items:
-                  $ref: "#/components/schemas/Passport"
-        "405":
+                $ref: "#/components/schemas/PassportResponse"
+        "400":
           description: "Invalid input"
     delete:
       tags:
-      - "passportStore"
-      summary: "Delete a passport"
-      description: ""
+        - "PassportStore"
+      summary: "Delete passport"
+      description: "Delete passport by ID"
       operationId: "deletePassport"
       parameters:
-      - name: "number"
-        in: "path"
-        description: "Passport number to delete"
-        required: true
-        schema:
-          type: "string"
+        - name: "id"
+          in: "path"
+          description: "Passport number to delete"
+          required: true
+          schema:
+            type: "string"
       responses:
         "200":
           description: "Successful operation"
           content:
-            '*/*' :
+            '*/*':
               schema:
-                type: object
-                items:
-                  $ref: "#/components/schemas/Passport"
+                $ref: "#/components/schemas/PassportResponse"
         "400":
           description: "Invalid name supplied"
         "404":
           description: "Passport not found"
 
+  /lostPassport:
```

Comment:
was added Lost passport which get a Person data because person couldn`t forget it data neither passport number